plugin.js

```diff
@@ -9,6 +9,9 @@ const webpack = require('webpack');
 const { SubresourceIntegrityPlugin } = require('webpack-subresource-integrity');
 const InjectPlugin = require('webpack-inject-plugin').default;
 
+// TODO: replace path usage in urls (prefix)
```

Comment:
No, it's actually something I wanted to look into. It's one of the two open issues mentioned in the original PR. Currently in main project this approach does not create hashes at all for external scripts, and this might be the cause. I'll try to look into this today. Will mark as draft until resolved.